src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -1101,7 +1102,41 @@ def test_ps_s3_notification_filter_on_master():
     result, status = s3_notification_conf.set_config()
     assert_equal(status/100, 2)
 
+    # create s3 to test type field - negative filter uptake
```

Comment:
would be great if you can split the negative filter test into its own test (mainly as the test is pretty heavy already).
ideally, tested against http or kafka endpoints (amqp endpoints are the hardest to setup, and also the least important...).
note that some tests are written in "endpoint agnostic" way, so they could eb tested agains all 3 endpoints types.

Reply:
Sure @yuvalif, will get to making it a separate test.

Reply:
didn't you move all the changes into their own test?